cratedb_toolkit/admin/xmover/analysis/shard.py

```
@@ -1,10 +1,11 @@
 """
 Shard analysis and rebalancing logic for CrateDB
 """
-
+import datetime
 import logging
 import math
 from collections import defaultdict
+from time import sleep
```

Ruff (0.12.2)

`time.sleep` imported but unused

Remove unused import: `time.sleep`

(F401)